.eslintrc

```diff
@@ -13,6 +13,7 @@
     "prefer-template": "error"
   },
   "settings": {
+    "import/parser": "babel-eslint",
```

Comment:
ah yeah this is a bad change. renaming the file is no good and adding additional config to it for the examples isn't worth it. the project doesn't use babel explicitly, so it's best to just ignore the offending example file in .eslintignore.